update examples for gorilla REPL

PilotTam opened the pull request.

hkulyc commented:
Didn't you say we should not have `lein uberjar` before? Why still keeping it in the doc?
Please try to name the files without space. See how examples in `src/` and [here](https://stackoverflow.com/questions/6709131/what-are-clojures-naming-conventions#:~:text=File%20names%20are%20lowercase%2C%20and,is%20clear%20and%20self%2Ddocumenting.).

PilotTam commented:
> Didn't you say we should not have `lein uberjar` before? Why still keeping it in the doc? Please try to name the files without space. See how examples in `src/` and [here](https://stackoverflow.com/questions/6709131/what-are-clojures-naming-conventions#:~:text=File%20names%20are%20lowercase%2C%20and,is%20clear%20and%20self%2Ddocumenting.).

Oh sorry I did not change the read me after reporting `lein uberjar` does not work. I deleted it. Also updated the .clj filenames with underscored. Thanks
